plugins/traefik/main.go

```diff
@@ -8,19 +8,32 @@ import (
 	"net"
 	"net/http"
 	"net/http/httptrace"
+	"os"
+	"strings"
+	"time"
 )
 
+type WebSocketEvent int
+
+const (
+	WebSocketRead WebSocketEvent = iota
+	WebSocketWrite
+	WebSocketClose
+)
+
+var wsEventChan = make(chan WebSocketEvent, 10)
```

Comment:
why 10? Why not 5, 16 or 11 :grin:

Reply:
tbh i thought i removed this value ^^ should be `nil`